### ISSUE TITLE
[BUG] Fix validation and metric code

### ISSUE THREAD

**Issue by chungSungMin** [boostcampaitech7/level2-cv-semanticsegmentation-cv-22-lv3#57]
# **어떤 문제점이 발생하였는지 작성해주세요**
- [x] 현재 validation에서 softmax를 사용하여 기본적으로 모델의 성능이 저하되는 현상을 겪고있음
- [x] Metric 또한 vaildation에 맞게 작성되어있어 수정이 필요해보임
<br><br>

# **수정한 할 파일들**
- [x] validation.py 에서 softmax를 사용해서 다른 클래스를 모두 버려 버리는 현상 떄문에 dice score가 낮게 나왔다고 생각하여 0.5롤 넘는 클래스를 모두 살리도록 sigmoid + 임계값을 설정하는 코드로 수정하였다.

- [x] vis_train 코드에서 원본 이미지 자체가 augmentation에서 정규화 되는 경우 모든 이미지가 검정색으로 나와 원본이미지를 시각화 하는 코드 자체는 삭제하였다.

- [x] trainer.py 에서 wandb log 찍힐떄 아래래와 같이 나와 f-string을 수정해주었다.
<img width="170" alt="image" src="https://github.com/user-attachments/assets/a4e99845-2741-40f3-9cb6-b359379fb772">

- [x] outputs = F.interpolate(outputs, size=(mask_h, mask_w), mode config.data.valid.interpolate.bilinear) 다음과 같이 이미지 보간을 하는 경우 선택하능한 보간법을 config에 정의하여 해당 이름만 바꿔주면 되도록 수정
 

<br><br>
